main/src/cgeo/geocaching/utils/DebugUtils.java

```diff
@@ -52,7 +70,11 @@ public static void createLogcat(@NonNull final Activity activity) {
                 if (Build.VERSION.SDK_INT < Build.VERSION_CODES.N) {
                     builder.command("logcat", "-d", "-f", file.getAbsolutePath());
                 } else {
-                    builder.command("logcat", "-d", "AndroidRuntime:E", "cgeo:D", "cgeo.geocachin:E", "*:S", "-f", file.getAbsolutePath());
+                    if (fullInfo) {
+                        builder.command("logcat", "-d", "*:V", "-f", file.getAbsolutePath());
+                    } else {
+                        builder.command("logcat", "-d", "AndroidRuntime:E", "cgeo:D", "cgeo.geocachin:E", "*:S", "-f", file.getAbsolutePath());
```

Comment:
Shouln`t we add info and warning for c:geo as well?

Reply:
The `fullInfo` does not exclude anything, the manual logs have the prefix "cgeo", and the system-written messages for c:geo have the package name as prefix (which is "cgeo.geocachin"), so everything c:geo related should be included.

Or have you found log messages prefixed by "c:geo" anywhwere?

Reply:
I probably misunderstood you - you're not talking about the prefix but about the log level?

Reply:
Exactly, I would suggest to add level warning and info for the c:geo package name to the normal logging.

Reply:
Added info/warning for package related logcat messages